.github/workflows/backend_integration_tests_pr.yml

```diff
@@ -2,17 +2,16 @@ name: CI
 
 on:
   pull_request:
-    branches:
-      - staging
     types:
       - opened
+      - ready_for_review
   pull_request_review:
     types:
       - submitted
 
 jobs:
   test:
-    if: ${{ github.event.review.state == 'APPROVED' || github.event_name == 'pull_request' }}
+    if: ${{ github.event.review.state == 'APPROVED' || (github.event_name == 'pull_request' && github.base_ref) }}
```

Comment:
Also fixed this to actually trigger the tests (`branches` refers to the source branch and not the target one)

Reply:
@AgustinRamiroDiaz shouldn't it be compared to some branch name?

Reply:
You are right! Fixed